.github/workflows/PublishTestResults.yml

```diff
@@ -0,0 +1,54 @@
+# ==============================================================================
+# Authors:
+#   Patrick Lehmann
+#   Unai Martinez-Corral
+#
+# ==============================================================================
+# Copyright 2020-2021 The pyTooling Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# SPDX-License-Identifier: Apache-2.0
+# ==============================================================================
+name: Publish Unit Test Results
+
+on:
+  workflow_call:
+
+jobs:
+  PublishTestResults:
+    name: 📊 Publish Test Results to Issue Comment
+    runs-on: ubuntu-latest
+    if: always()
+
+    steps:
+      - name: ⏬ Checkout repository
+        uses: actions/checkout@v2
+
+      - name: Download Artifacts
+        uses: actions/download-artifact@v2
+        with:
+          path: artifacts
+
+# TODO: @umarcor do we want to make it optional or just kick it out?
```

Comment:
I personally don't find that implementation particularly exciting. I believe it is good enough with dorny/test-reporter. In practice, I would expect most users of the Pipeline to replace this job with their specific reporting/hook needs. Therefore, this is mostly for us and for people who want something that just works.

Reply:
Removed in c2f7d273.